webbpsf/opds.py

```diff
@@ -444,63 +451,69 @@ def label_seg(self, segment, ax=None, show_axes=False, color='black', pupil_orie
 
         if ax is None:
             ax = plt.gca()
-        label = ax.text(cx + offset, (cy + offset)*ysign, segment, color=color, horizontalalignment='center', verticalalignment='center')
+        label = ax.text(cx + offset, (cy + offset) * ysign, segment, color=color, horizontalalignment="center", verticalalignment="center")
 
         if show_axes:
-            ax_arrow_len = .3
+            ax_arrow_len = 0.3
             # if not ('C' in segment ):
             if True:
-
-                for i, color, label in zip([0, 1, 2], ['green', 'blue', 'red'], ['x', 'y', 'z']):
+                for i, color, label in zip([0, 1, 2], ["green", "blue", "red"], ["x", "y", "z"]):
                     vec = np.matrix([0, 0, 0]).transpose()  # xyz order
                     vec[i] = 1
                     b = self._rot_matrix_local_to_global(segment) * vec
                     b = np.asarray(b).flatten()  # Inelegant but it works
 
-                    ax.arrow(cx, cy*ysign, ax_arrow_len * b[0], (ax_arrow_len * b[1])*ysign, color=color,
-                             # width=ax,
-                             head_width=.050, head_length=.080)  # in units of mm
+                    ax.arrow(
+                        cx,
+                        cy * ysign,
+                        ax_arrow_len * b[0],
+                        (ax_arrow_len * b[1]) * ysign,
+                        color=color,
+                        # width=ax,
+                        head_width=0.050,
+                        head_length=0.080,
+                    )  # in units of mm
 
                     xoffset = 0.1 if i == 2 else 0
-                    ax.text(cx + ax_arrow_len * b[0] * 1.5 + xoffset, (cy + ax_arrow_len * b[1] * 1.5)*ysign, label,
-                            color=color, fontsize=8,
-                            horizontalalignment='center', verticalalignment='center'
-                            )
+                    ax.text(
+                        cx + ax_arrow_len * b[0] * 1.5 + xoffset, (cy + ax_arrow_len * b[1] * 1.5) * ysign, label, color=color, fontsize=8, horizontalalignment="center", verticalalignment="center"
+                    )
```

Comment:
I also don't understand about the line length choices. As I've said, I'm not a fan of 80 character line limits, but here it's taken code and compressed it to one line, which as a result ends up being like 180 characters long. That seems a bit excessive... What do you think?   

Was there any specific line length limit value set when using ruff for this?